.pylintrc

```diff
@@ -0,0 +1,694 @@
+# Pylint 3.2.x configuration file
```

Comment:
you want to include this pylintrc? seems fairly specific.

Reply:
I want to include it so that there is consistency when I make future changes. Happy to include OpenRelik specific pylintrc.